tapiro/core/src/main/scala/io/buildo/tapiro/Cli.scala

```diff
@@ -36,9 +36,10 @@ object Cli {
 
     OParser.parse(parser, args, CliConfig()) match {
       case Some(c) => {
-        c.`package`.split(".").toList match {
+        c.`package`.split("\\.").toList match {
```

Comment:
This is unrelated, but I was using the CLI to test and it seems `.` should be escaped here because it's used as a regex